opm/parser/eclipse/EclipseState/Schedule/Completion.hpp

```diff
@@ -55,6 +58,10 @@ namespace Opm {
         double getDiameter() const;
         double getSkinFactor() const;
         void   fixDefaultIJ(int wellHeadI , int wellHeadJ);
+        int getSegmentNumber() const;
+        void setSegmentNumber(const int segment_number);
+        double getCenterDepth() const;
+        void setCenterDepth(const double center_depth);
```

Comment:
I think the two methods `setSegmentNumber()` and `setCenterDepth()` could be combined to one method:
```C++
void attachSegment( int segmentNumber , double centerDepth) {
    m_segmentNumber = segmentNumber;
    m_center_depth = centerDepth;
}
```